modules/ROOT/pages/customize.adoc

```diff
@@ -18,7 +18,7 @@ theme menu, page structure menu, and settings menu.
 
 == Pages List
 
-The *Pages* list shows the pages included with ACM and the pages you have added,
+The *Pages* list shows the pages included with API Community Manager and the pages you have added,
```

Comment:
This is referred to as "the Pages menu" above?

Reply:
Good catch. I updated the earlier reference to match.